src/inline_snapshot/_config.py

```diff
@@ -48,6 +50,13 @@ def read_config(path: Path) -> Config:
                 "shortcuts", {"fix": ["create", "fix"], "review": ["review"]}
             )
 
+            if snapshot_dir := config.get("snapshot-dir"):
```

Comment:
```suggestion
            if snapshot_dir := config.get("snapshot-dir",".inline-snapshot/external"):
```
This would simplify code in pytest_plugin.py

Reply:
can we also use a different name for `snapshot-dir`? maybe something like `external-hash-storage`?

snapshots are still stored in the source code :-) and I want to provide different ways to store them in the future, using something with `hash` in the name would make sense do differentiate it.

Reply:
> This would simplify code in pytest_plugin.py

Unfortunately this isn't the same thing: this would write external files *next to pyproject.toml* by default, instead of *next to pytest.ini* (current behavior). If you think this is OK (few to no users relying on that), then we can apply this suggestion. I'm myself indifferent since I'll use the option to change the directory anyway.

> can we also use a different name for snapshot-dir? maybe something like external-hash-storage?
snapshots are still stored in the source code :-) and I want to provide different ways to store them in the future, using something with hash in the name would make sense do differentiate it.

IMO "snapshot directory" is more intuitive, as the actual snapshot value is what's written in the external file, while in code you get a *pointer* (`external`). The documentation update should explain that better though, true. In any case, I don't have a strong opinion on this, so if you prefer `external-hash-storage`, I'm happy to go with it :slightly_smiling_face:

Reply:
> If you think this is OK (few to no users relying on that), then we can apply this suggestion

Ok, then ignore my suggestion.

> IMO "snapshot directory" is more intuitive, 

What do you think about making `.inline-snapshot/` the default for `snapshot-dir`?
inline-snapshot will then use a `external/` folder inside this directory

The point is, that we might have different ways to store external snapshots in the future (uuid, hash, ...), which maybe need different folders where external snapshots are stored.

Having one folder where inline-snapshot can store different things is really useful in this case.

Reply:
Sure, sounds good, but shouldn't it happen in another PR? Currently inline-snapshot just sets the global `storage` variable with a directory path. Changing the system so that inline-snapshot uses different subfolders within the storage folder is kind of a bigger change which I'm not really comfortable tackling in this PR :sweat_smile: Unless I misunderstood what you meant?